rep-2000.rst

```diff
@@ -209,18 +209,20 @@ Crystal Clemmys (December 2018 - December 2019)
 Targeted platforms:
 
 +--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
-| Architecture | Ubuntu Bionic (18.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2017)  | Ubuntu Xenial (16.04) [s] | Debian Stretch (9) [s] |
+| Architecture | Ubuntu Bionic (18.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2017)  | Ubuntu Xenial (16.04)     | Debian Stretch (9)     |
 +==============+=======================+==============================+========================+===========================+========================+
-|    amd64     |        Tier 1         |           Tier 1*            |          Tier 1*       |           Tier 2          |        Tier 3          |
+|    amd64     |  Tier 1 [d][a][s]     |           Tier 1 [a][s]      |          Tier 1 [a][s] |           Tier 2 [s]      |        Tier 3  [s]     |
 +--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
-|    arm64     |        Tier 1         |                              |                        |           Tier 2          |        Tier 3          |
+|    arm64     |  Tier 1 [d][a][s]     |                              |                        |           Tier 2  [s]     |        Tier 3 [s]      |
 +--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
 
+The following indicators show what delivery mechanisms are available for each platform.
 
-" * " Individual binary packages are not currently available for these platforms.
-Binary releases are provided as a single archive per platform containing all packages in [this repos file](https://github.com/ros2/ros2/blob/master/ros2.repos).
+" [d] " Debian packages will be provided for this platform for packages submitted to the rosdistro.
 
-" [s] " Compilation from source, the ROS buildfarm will not produce any binary packages for these platforms.
+" [a] " Binary releases are provided as a single archive per platform containing all packages in this repos file [4]_. 
```

Comment:
The repos link for Crystal and Dashing should be different from each other and reference the distro specific version of the file.